app/controllers/bank_types_controller.rb

```diff
@@ -0,0 +1,53 @@
+class BankTypesController < ApplicationController
+  load_and_authorize_resource
+
+  before_action :load_company
+  before_action :load_data, only: [:index, :new, :create, :edit]
+  before_action :data_bank_type, only: [:create, :update, :destroy]
```

Comment:
should change name to : load_bank_type_data
And I think only : index. create, update, destroy